python/paddle/fluid/layers/nn.py

```diff
@@ -11322,7 +11322,7 @@ def scale(x, scale=1.0, bias=0.0, bias_after_scale=True, act=None, name=None):
     return helper.append_activation(out)
 
 
-def elementwise_add(x, y, axis=-1, act=None, name=None):
+def elementwise_add(x, y, axis=-1, act=None, name=None, use_mkldnn=False):
```

Comment:
The fate of this line shall be decided after we resolve the issue discussed here: https://github.com/PaddlePaddle/Paddle/pull/25713#discussion_r461587882